src/main/java/ro/societateahermes/backendservice/entities/User.java

```diff
@@ -24,4 +30,24 @@
     private String field;
     private String language;
 
+    @OneToMany(mappedBy = "user")
+    private List<Participation> listOfParticipation;
+
+
+    public User(String firstName,String lastName,String email,String username,String password,String university)
+    {
+        this.firstName=firstName;
+        this.lastName=lastName;
+        this.email=email;
+        this.username=username;
+        this.password=password;
+        this.university=university;
+    }
+
+    public void addParticipation(Participation participation)
```

Comment:
Metoda asta nu are ce sa caute in entitate. Locul ei este intr-un userService, unde se va lua lista din entitate si i se va adauga din service participarile